app.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express")
+const cors = require("cors")
+const app = express();
+const bodyParser = require("body-parser")
+const port =3000;
+const school = require("./School.js")
+app.use(cors())
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({extended:false}))
+
+let newSchool = new school ()
+// console.log(newSchool)
+
+app.get("/school",(req,res)=>{
+    res.json(newSchool.classes)
+})
+
+app.post("/school/add/class",(req,res)=>{
+    try{
+        newSchool.addClass(req.query.name,req.query.teacher)
+        res.json({ 
+            "class": { "name": req.query.name, "teacher": req.query.teacher, "students": []},
+            "message": "Created a new class",
+            "timestamp": timeStamp()
+          })
+    }catch(err){
+        res.json(
+            { "error": err +"Please fill out all the information or Class already exists",
+            "timestamp": timeStamp()
+        }
+        )
+    }
+})
+app.post("/school/add/student/:class",(req,res)=>{
```

Comment:
`/school/<class-name>/enroll` 

All these endpoints were specified on the readme doc